rgw/v2/utils/utils.py

```diff
@@ -645,6 +645,26 @@ def get_realm_source_zone_info():
     return realm, source_zone
 
 
+def get_zonegroup_name():
+    op = exec_shell_cmd("radosgw-admin sync status")
+    lines = list(op.split("\n"))
+    for line in lines:
+        if "zonegroup" in line:
+            zonegroup_name = line[line.find("(") + 1 : line.find(")")]
+            break
+    return zonegroup_name
+
+
+def get_zone_name():
+    op = exec_shell_cmd("radosgw-admin sync status")
+    lines = list(op.split("\n"))
+    for line in lines:
+        if "zone " in line:
+            zone_name = line[line.find("(") + 1 : line.find(")")]
+            break
+    return zone_name
+
+
```

Comment:
Please remove one method definition and pass zone or zonegroup as parameter

Reply:
updated the changes